common/source/docs/common-cabling-guide.rst

```diff
@@ -0,0 +1,275 @@
+
+- Keep cables as short as possible.
+- Use twisted pairs to reduce crosstalk:
+  - Twist each pair (SCL/+5V and SDA/GND) about 10 turns per 30 cm.
+  - Twist both pairs together about 4 turns per 30 cm.
+- For larger vehicles or long cables, consider using CAN bus or other differential signaling interfaces instead.
+
+Pull-up Resistors
++++++++++++++++++
+
+Pull-up resistors on both SDA and SCL lines are essential. Autopilots or peripheral devices typically have these resistors built-in. An oscilloscope measurement is sometimes required to check correct value of pull-up resistors. If the signal amplitude is too low, increase resistor values. If the signals are rounded, decrease resistor values.
+
+Common Problems
++++++++++++++++
+
+- **Address Clashes**: Occur when multiple devices on the bus share the same address, preventing proper communication. Possible solutions include:
+  - Changing the device address via hardware or software configuration, if supported.
+  - `Using address translator <https://docs.thunderfly.cz/avionics/TFI2CADT01/>`_, which allows multiple devices with identical addresses to coexist by remapping their addresses.
+
+- **Excessive Wiring Capacitance**: Long cables or multiple devices connected to a single bus significantly increase cable capacitance, resulting in degraded signal quality. Possible solutions include:
+  - Reducing cable length or using higher-quality cables.
+  - Splitting devices across multiple I2C buses.
+  - `Employing I2C bus accelerators or extenders <https://docs.thunderfly.cz/avionics/TFI2CEXT01/>`_, to boost signal quality over extended cable lengths.
+
+I2C Bus Accelerators and Translators
+++++++++++++++++++++++++++++++++++++
+
+- **I2C Bus Accelerators** (e.g., `ThunderFly TFI2CEXT01 <https://docs.thunderfly.cz/avionics/TFI2CEXT01/>`_) physically divide the bus into segments, amplifying signals and reducing the impact of wiring capacitance, thus improving signal integrity and reliability on longer cables. The TFI2CEXT01 can also perform voltage level translation between 3.3V and 5V logic levels.
+- **I2C Address Translators** (e.g., `ThunderFly TFI2CADT01 <https://docs.thunderfly.cz/avionics/TFI2CADT01/>`_) allow multiple identical devices with the same I2C address to be connected by remapping device addresses dynamically, resolving conflicts and simplifying sensor integration.
+
+CAN
+---
+
+:ref:`CAN <common-canbus-setup-advanced>` bus is provided on many autopilots for use with :ref:`DroneCAN <common-uavcan-setup-advanced>` peripherals. ESCs, GPS/Compass, Rangefinders, and many other peripherals and sensors are being added to the list of available DroneCAN devices everyday. CAN provides a robust method of communicating with peripherals with data integrity, even with long leads.
+
+
+Signals
++++++++
+
+Power and ground lines are typically provided alongside the CAN signals on standard 4-pin connectors (e.g., JST-GH).
+
+.. list-table:: CAN Signal Colour Examples
```

Comment:
same comment about this table also...really adds no value to the general color code table already listed above

Reply:
Thanks! Similar to I2C, this table reflects real-world connector layouts and manufacturer-specific cable coloring. It helps distinguish systems visually.